mdata/store_cassandra.go

```diff
@@ -362,7 +366,12 @@ func (o asc) Less(i, j int) bool { return o[i].sortKey < o[j].sortKey }
 
 func (c *CassandraStore) processReadQueue() {
 	for crr := range c.readQueue {
-		cassGetWaitDuration.Value(time.Since(crr.timestamp))
+		waitDuration := time.Since(crr.timestamp)
+		cassGetWaitDuration.Value(waitDuration)
+		if waitDuration > c.omitReadTimeout {
```

Comment:
hmm, maybe not.  But if the waitDuration is timed from when the search is started until when it is pulled from the queue, then why were we seeing waitDurations if a few seconds but queries still be queued and running for 10+minutes after requests stopped being sent to a cluster.